src/DevBetterWeb.Web/Startup.cs

```diff
@@ -114,6 +115,10 @@ public void ConfigureServices(IServiceCollection services)
 
     services.AddScoped<IVideosService, VideosService>();
     services.AddSingleton<IVideosCacheService, VideosCacheService>();
+
+    services.AddScoped<WebVTTParsingService>();
+    services.AddScoped<VideoDetailsService>();
```

Comment:
If interfaces are applied.
```suggestion
    services.AddScoped<IWebVTTParsingService, WebVTTParsingService>();
    services.AddScoped<IVideoDetailsService, VideoDetailsService>();
```